package-lock.json

```diff
@@ -1,34 +1,181 @@
 {
```

Comment:
I see quite a few new libraries being added in the lock file. This shouldn't be needed, from what I see there are no new dependencies that the library requires. Could we revert this file?

Reply:
I used "var request = require('request-promise-native');" to get the response headers